x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/pipeline_select_option.tsx

```diff
@@ -7,107 +7,70 @@
 
 import React from 'react';
 
-import {
-  EuiFlexGroup,
-  EuiFlexItem,
-  EuiIcon,
-  EuiSpacer,
-  EuiText,
-  EuiTextColor,
-  EuiTitle,
-} from '@elastic/eui';
-import { i18n } from '@kbn/i18n';
+import { EuiFlexGroup, EuiFlexItem, EuiIcon, EuiText, EuiTextColor, EuiTitle } from '@elastic/eui';
 
 import { MLModelTypeBadge } from '../ml_model_type_badge';
 
 import { MLInferencePipelineOption } from './ml_inference_logic';
 import { EXISTING_PIPELINE_DISABLED_MISSING_SOURCE_FIELDS, MODEL_REDACTED_VALUE } from './utils';
 
 export interface PipelineSelectOptionProps {
+  checked?: 'on';
+  disabled?: boolean;
+  label: string;
   pipeline: MLInferencePipelineOption;
 }
 
+// TODO: Make disabledReason required and remove EXISTING_PIPELINE_DISABLED_MISSING_SOURCE_FIELDS call without args
 export const PipelineSelectOptionDisabled: React.FC<{ disabledReason?: string }> = ({
   disabledReason,
 }) => {
   return (
-    <>
-      <EuiSpacer size="xs" />
+    <EuiFlexGroup alignItems="center" gutterSize="s" responsive={false}>
+      <EuiFlexItem grow={false}>
+        <EuiIcon type="warning" color="warning" />
+      </EuiFlexItem>
       <EuiFlexItem>
-        <EuiFlexGroup alignItems="center" gutterSize="s">
-          <EuiFlexItem grow={false}>
-            <EuiIcon type="warning" color="warning" />
-          </EuiFlexItem>
-          <EuiFlexItem>
-            <EuiTextColor color="warning">
-              {disabledReason ?? EXISTING_PIPELINE_DISABLED_MISSING_SOURCE_FIELDS}
-            </EuiTextColor>
-          </EuiFlexItem>
-        </EuiFlexGroup>
+        <EuiTextColor color="warning">
+          {disabledReason ?? EXISTING_PIPELINE_DISABLED_MISSING_SOURCE_FIELDS}
+        </EuiTextColor>
       </EuiFlexItem>
-      <EuiSpacer size="xs" />
-    </>
+    </EuiFlexGroup>
   );
 };
 
 export const PipelineSelectOption: React.FC<PipelineSelectOptionProps> = ({ pipeline }) => {
   const modelIdDisplay = pipeline.modelId.length > 0 ? pipeline.modelId : MODEL_REDACTED_VALUE;
   return (
-    <EuiFlexGroup direction="column" gutterSize="none">
+    // TODO: Add model state & pipeline info link. Make sure to check mobile rendering when doing this!
+    <EuiFlexGroup direction="column" gutterSize="xs">
+      <EuiFlexItem>
+        <EuiTitle size="xxs">
+          <h4>{pipeline.pipelineName}</h4>
+        </EuiTitle>
+      </EuiFlexItem>
       <EuiFlexItem>
-        <EuiFlexGroup gutterSize="s" alignItems="center" justifyContent="flexEnd">
-          <EuiFlexItem>
-            <EuiTitle size="xs">
-              <h4>{pipeline.pipelineName}</h4>
-            </EuiTitle>
+        <EuiFlexGroup gutterSize="s" justifyContent="flexStart">
+          <EuiFlexItem grow={pipeline.modelType.length <= 0}>
```

Comment:
Can a string's length be negative? 🙂  If not:
```suggestion
          <EuiFlexItem grow={pipeline.modelType.length === 0}>
```